package.json

```diff
@@ -11,14 +11,17 @@
     "scripts": {
         "postinstall": "pnpm build",
 
-        "build": "pnpm build:eslint && pnpm build:tsup && pnpm --filter ./packages/* -r build",
-        "build:eslint": "pnpm --filter=@workleap/eslint-plugin build",
-        "build:tsup": "pnpm --filter=@workleap/tsup-configs build",
+        "build": "pnpm build:tsup && pnpm build:swc && pnpm build:eslint && pnpm build:stylelint && pnpm --filter=!@workleap/tsup-configs --filter=!@workleap/swc-configs --filter !@workleap/eslint-plugin --filter !@workleap/stylelint-config -r build",
```

Comment:
should samples be built to make sure tsup config is working properly?

Reply:
Could you clarify I don't understand how that would affect tsup?

Furthermore I prefer to not penalize the build of the packages with the sample build time.

Reply:
The goal of the sample app is to have a place where we can test the config and make sure everything works. 
- We run pnpm test on the sample, making sure the test in there are valid, that the swc jest config is valid and everything works properly. 
- We run linting styles and eslint to make sure everything works properly on the sample as well
However,
- We do not run typechecking on the sample, so if a mistake in the typings occurs, we won't be notified during the build
- We do not try to build the components packages that uses the tsup config, so if something is broken we won't be notified.

There is an opportunity there to run the "build" process, validating both the type checking and tsup building, to make sure that everything works properly

Reply:
The CI build the sample app

![image](https://github.com/workleap/wl-web-configs/assets/794579/263f07c0-5660-43e3-b71b-5ce9446b24c0)

Reply:
And yes we do lint the sample as the packages are part of the workspace and we do not filter any packages (except the ts configs packages for the types)

```
        "lint:eslint": "eslint . --max-warnings=1 --cache --cache-location node_modules/.cache/eslint",
        "lint:stylelint": "stylelint \"**/*.css\" --cache --cache-location node_modules/.cache/stylelint",
        "lint:types": "pnpm -r --parallel --filter=!@workleap/typescript-configs exec tsc",
```

Reply:
Jest include the sample packages:

![image](https://github.com/workleap/wl-web-configs/assets/794579/ec2c8b9d-457c-46c9-91f3-b7f43844d16f)